setup.py

```diff
@@ -110,7 +110,7 @@ def download_mafft(kind="deb") -> Path:
 
 setup(
     name="seqlike",
-    version="1.1.6",
+    version="1.1.7.dev1",
```

Comment:
```suggestion
    version="1.1.6",
```

We have a GitHub Action for cutting releases that depend on having a sync'd up version number with the appropriate configuration files. Therefore, I'd like to request that we change this back to 1.1.6.

Also, while I appreciate the sentiment for having dev releases, until I learn how to configure bump2version to include dev as part of the version (and thus update the appropriate GitHub action that we have), we're probably going to stick with major/minor/patch as our convention for now. That can be revisited in the future, though.

Reply:
Thank you, for explaining.